exosphere-shared/templates/add-service/exoservice-ls-mongodb/features/support/world.ls

```diff
@@ -6,7 +6,7 @@ require! {
 World = !->
 
   # Fills in _____modelName_____ ids in the placeholders of the template
-  @fill-in-_____modelName_____-ids = (template, done) ->
+  @fill-in-modelName-ids = (template, done) ->
```

Comment:
Sorry about that. I didn't know I wasn't supposed to run the tests. I've undone those changes.